gsc/test/suse-bash.dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG BUILD_OS
+
+From $BUILD_OS
+
+ARG BUILD_OS
+
+RUN if [ "$BUILD_OS" = "registry.suse.com/suse/sle15:15.4" ] || [ "$BUILD_OS" = "opensuse/leap:15.6" ]; then \
```

Comment:
If this file is only called for suse os, then why build os comparison

Reply:
Updated